allennlp/commands/fine_tune.py

```diff
@@ -10,7 +10,7 @@
 import logging
 import os
 from copy import deepcopy
-
+import re
```

Comment:
nit: I like to leave blank lines between the grouped imports: (standard library) -> (external libraries) -> (allennlp modules)

Reply:
Ohh!! I didn't realise the purpose of blanks...